uktrade/great-cms
# Update signup to resend verification code is current one has expired

This commit modifies the sign up flow so that if an expired verification code is entered by the user then a new code is sent to them. They are also presented with a message in the UI to inform them that they've been sent a new code.

To test
* Create a new user account in great-cms
* Wait 30 minutes for verification code to expire*
* Enter expired verification code
* Check to see if new verification code has been sent
* Enter new verification code

\* If testing in local environment then sso/verification/models.py in directory-sso can be modified to reduce the code expiry time.

Before and after for UI changes:
![Screenshot 2022-08-19 at 09 48 01](https://user-images.githubusercontent.com/7093347/185582802-4427ccb7-9284-4623-94ac-4b6130855a21.png)
![Screenshot 2022-08-19 at 09 34 04](https://user-images.githubusercontent.com/7093347/185582674-41161825-c9ac-4c8d-a01a-4e9a0c0740ac.png)

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GLS-336

### Reviewing help


### Housekeeping


### Merging
